From: Askar Safin <safinaskar@gmail.com>
Subject: .get_maintainer.ignore: add Askar
Date: Wed, 1 Apr 2026 07:46:19 +0000

I don't want get_maintainer.pl to automatically print my email.

Link: https://lkml.kernel.org/r/20260401074619.988459-1-safinaskar@gmail.com
Signed-off-by: Askar Safin <safinaskar@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 .get_maintainer.ignore |    1 +
 1 file changed, 1 insertion(+)

--- a/.get_maintainer.ignore~get_maintainerignore-add-myself
+++ a/.get_maintainer.ignore
@@ -1,6 +1,7 @@
 Alan Cox <alan@lxorguk.ukuu.org.uk>
 Alan Cox <root@hraefn.swansea.linux.org.uk>
 Alyssa Rosenzweig <alyssa@rosenzweig.io>
+Askar Safin <safinaskar@gmail.com>
 Christoph Hellwig <hch@lst.de>
 Jeff Kirsher <jeffrey.t.kirsher@intel.com>
 Marc Gonzalez <marc.w.gonzalez@free.fr>
_
